src/Cake.Svn/SvnAliases.Add.cs

```diff
@@ -1,51 +1,113 @@
 ﻿using Cake.Core;
 using Cake.Core.Annotations;
+using Cake.Core.IO;
 using Cake.Svn.Add;
 
 namespace Cake.Svn
 {
     public static partial class SvnAliases
     {
         /// <summary>
-        /// Add a file or directory to Subversion.
+        /// Add a directory to Subversion.
         /// </summary>
         /// <param name="context">The Cake context.</param>
-        /// <param name="fileOrDirectoryToAdd">The filename or directoryname to add.</param>
-        /// <returns>True if the command were successful. Otherwise False will be returned.</returns>
+        /// <param name="directory">The directory.</param>
+        /// <returns>
+        /// <c>true</c> if the command were successful. Otherwise <c>false</c> will be returned.
+        /// </returns>
         /// <example>
-        /// <para>Add a file or a directory to Subversion.</para>
+        /// <para>Add a directory to Subversion.</para>
+        /// <code>
+        /// <![CDATA[ 
+        ///     var directoryAdded = SvnAdd(
+        ///         new DirectoryPath(@"C:\project\src\"));
+        ///
+        ///     Verbose("Directory added: {0}", directoryAdded);
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Add")]
+        [CakeNamespaceImport("Cake.Svn.Add")]
+        public static bool SvnAdd(this ICakeContext context, DirectoryPath directory)
+        {
+            return SvnAdd(context, directory, null);
+        }
+
+        /// <summary>
+        /// Add a directory to Subversion.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="directory">The directory.</param>
+        /// <param name="settings">The settings.</param>
+        /// <returns>
+        /// <c>true</c> if the command were successful. Otherwise <c>false</c> will be returned.
```

Comment:
`Returns <c>true</c> if the command was successful. Otherwise <c>false</c> will be returned.`